frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -110,7 +110,23 @@ function ReaderHighlight:genHighlightDrawerMenu()
     }
 end
 
-function ReaderHighlight:clear()
+function ReaderHighlight:getClearTag()
+    -- Returns a tag, that can be provided on delayed call to clear(tag)
```

Comment:
I'm not sure I'm sold on this "tag" name as opposed to something like random_id or highlight_time. In this context it kind of makes me think of HTML tags. But the reason I dropped in is to ask why you didn't this above the function? :-)

Reply:
Why I didn't _what_ this above this function? If what = "added this comment above the function", as a docstring, there are none in this file, and I didn't want this one about a not-high-level function to be the single one :)
I could use Flag instead of Tag (but flag feels boolean, tag for me feels like any sticker, id, string... anything that changes and identify something).
Or clear_id and getClearId() is that's fine with you.

Reply:
Sure, let's go with clear_id.

Not sure what happened with that sentence. I think I wrote "why didn't you put this above the function?" and then edited it to add the rest. :-P

> I didn't want this one about a not-high-level function to be the single one :)

You can always just stick it in there with an insufficient number of dashes (i.e., the exact same comment as now with two dashes, just above rather than inside). But actually until the module gets a top comment I don't think it'd make a difference anyway.